x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/index.tsx

```diff
@@ -97,10 +107,14 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn, 'field
           column.params.orderBy.type === 'alphabetical' ? '_key' : column.params.orderBy.columnId,
         order: column.params.orderDirection,
         size: column.params.size,
-        otherBucket: false,
-        otherBucketLabel: 'Other',
-        missingBucket: false,
-        missingBucketLabel: 'Missing',
+        otherBucket: Boolean(column.params.otherBucket),
+        otherBucketLabel: i18n.translate('xpack.lens.indexPattern.terms.otherLabel', {
+          defaultMessage: 'Other',
+        }),
+        missingBucket: column.params.otherBucket && column.params.missingBucket,
+        missingBucketLabel: i18n.translate('xpack.lens.indexPattern.terms.missingLabel', {
+          defaultMessage: 'Missing',
```

Comment:
That's a design comment, not a dev one – I feel like some users could not clearly see the difference between 'other' and 'missing'. On the other hand, we have a popover in a flyout and I am not sure we want to add a tooltip on the top of it with explanation...

Reply:
What about changing it to `(missing field)` or `(missing value)`? It might be more clear than just the plain text?